tests/functional/Computer.php

```diff
@@ -251,14 +268,21 @@ public function testCreateLinks()
     {
         global $CFG_GLPI;
 
+        $this->login();
+
         $computer = $this->getNewComputer();
         $saveconf = $CFG_GLPI;
```

Comment:
With your changes, saving the config and restoring it at the end of tet is now useless. You should clean the related code.